Assets/Plugins/iOS/EOSSDK.framework/Info.plist

```diff
@@ -41,7 +41,7 @@
 	<key>DTXcodeBuild</key>
 	<string>9M174d</string>
 	<key>MinimumOSVersion</key>
-	<string>11.0</string>
+	<string>12.0</string>
```

Comment:
Does this change imply that a change needs to be made with respect to our documentation?

Reply:
I don't see anything in our documentation that references "11" or a minimum OS Version. Our MacOS documentation doesn't mention OS versions.

Should it? Maybe this is something to document?

Reply:
Noted in #EOS-2184